modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/paths/TestrayResults.path

```diff
@@ -35,6 +35,11 @@
 	<td>//span[normalize-space()='${key_status}']</td>
 	<td></td>
 </tr>
+<tr>
+	<td>TOTAL_TEST_CASE_RESULTS</td>
+	<td>//button/div/h5[text()='Total test cases']//following::*[text()='${numberTotal}']</td>
+	<td></td>
+</tr>
```

Comment:
@joelrocha68 would you please set the xpath to:
//button//h5[text()='Total test cases']//following::*[text()='${numberTotal}']

The less we specify the tags, the lesser the probability to break after a dev change. We need to find balance with it